### ISSUE TITLE
Bundle together multiple pipelines for a single entity into a single endpoint

### ISSUE THREAD

**Issue by jde** [coralproject/atoll#35]
__Challenge__: We need to be able to get everything we can know about a single Entity via a single api call. The output of this operation will be appended to the Entity document in the database.

Our initial Entities are _users_, _comments_ and _assets_.

__Spec__: 
* Receive post input of a single comment, user or asset and related data.
* Feed input through a series of pipelines for that Entity type: https://github.com/coralproject/atoll/issues/36
* Return an object to be appended to that Document in the database.

**Comment by jde** [comment]
Sorry for the confusion (and the late answer here although I believe we discussed it offline).  I see that my wording has been conflating two things:

We're looking for metrics _on_ individual entities, aka values that we can attach to a single document.  We should take advantage of all the data that we have across the database to find those values.

I updated the issue you quoted above to make this more clear.
